python/tskit/trees.py

```diff
@@ -5522,6 +5522,44 @@ def genetic_relatedness(
         :ref:`polarised <sec_stats_polarisation>`,
         and :ref:`return value <sec_stats_output_format>`.
 
+        What is computed depends on ``mode``:
+
+        "site"
+            Number of pairwise allelic matches in the window between two
+            sample sets relative to the rest of the sample sets. To be precise,
+            let `m(a,b)` denote the total number of pairwise allelic matches
+            between sample sets `a` and `b` and let `C` and `D` be
+            independently chosen sample sets. Then, this computes
+            `E[m(a,b) - m(a,C) - m(b,D) + m(C,D)]`.
+            This can also be seen as the covariance of a quantitative trait
+            corresponding to two sample sets (or "individuals"). Let each
+            allele be associated with an effect drawn from a `N(0,1/2)`
+            distribution, and let the trait value of a sample set be the sum of
+            its allele effects. Then, this computes covariance between the
+            trait values of two sample sets. For diploid sample sets, when
```

Comment:
And then "For example, to compute covariance between traits of diploid individuals, each sample set would be the pair of genomes of one individuals (and in this case, if ``proportion=True``, this corresponds to ...)."